scss/titles.scss

```diff
@@ -0,0 +1,15 @@
+.title {
+  align-items: center;
+  display: flex;
+  margin: 10px;
+  position: relative;
+
+  p {
+    color: $black;
+    font-family: 'Roboto', sans-serif;
+    font-size: 40px;
```

Comment:
you should use the sizes defined in styles.scss ($text-big, $text-medium, $text-small)